duckduckgo/Android
# Unsupported email screen for devices where we cannot encrypt data

Task/Issue URL: https://app.asana.com/0/1125189844152671/1200641147283978
Tech Design URL: 
CC: 

**Description**:
In some edge cases we are not able to encrypt email data. For these devices we now show a not supported screen instead of Email Protection

**Steps to test this PR**:
1. Inside `EmailEncryptedSharedPreferences`, in the `encryptedPreferences()` method change the code so it always returns null.
1. Go to settings and click on the email feature
1. The UI should show the not supported UI
1. Uncomment the code from step 1
1. Go to settings and click on the email beta setting
1. The UI should be the normal one.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
